README.md

```diff
@@ -159,6 +159,7 @@ Table of Contents
   * [Clarifai](https://www.clarifai.com) — Image API for custom face recognition and detection. Able to train AI models. Free plan has 5000 calls per month.
   * [Colaboratory](https://colab.research.google.com) — Free web-based Python notebook environment with Nvidia Tesla K80 GPU.
   * [CurrencyScoop](https://currencyscoop.com) - Realtime currency data API for fintech apps. Free plan includes 5000 calls per month.
+  * [Dataflow Kit](https://dataflowkit.com/) — Web scraping and data extraction AP. Turn any website into usuful API. Automate data workflows on the web, process, and transform data at any scale. Free 1000 credits after registration.
```

Comment:
`Once you sign-up, we grant you free 1000 credits, which is equal to €5 for evaluation and testing. `

This is not a free tier, its a short preview account, I am afraid that would not satisfy the requirements for this list

Reply:
Thank you for your answer. 
I've just check out several random API providers from the list and found none of them offer free tier anymore. 
ScrapingAnt, Scraper API,  ScrapingNinja

Reply:
When they were added they had free tiers, if they do not now it would be great if you could send a PR to remove them.  That's how the list is maintained, we can't actively verify this in an ongoing manner - its a collaborative effort.